docs/app/Examples/Button/Animated.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <Button className='animated'>
+        <div className='visible content'>Next</div>
+        <div className='hidden content'>
+          <i className='right arrow icon'></i>
```

Comment:
Also strange the linter is not catching this, I'll look into it.  Elements with no body should be self closing:

```jsx
<i className='right arrow icon' />
```

Instead of:
```jsx
<i className='right arrow icon'></i>
```